governance/votes/Executive vote - July 4, 2024.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Template - [Executive Vote] Out-of-Schedule Executive Vote - Approve LITE-PSM-USDC DAO Resolution - July 04, 2024
+summary: Approving a DAO Resolution pertaining to the existing Coinbase Custody arrangement (RWA014-A) and the new Lightweight Peg Stability Moodule (LITE-PSM-A). 
```

Comment:
I would specify what the DAO Resolution is defining. "Pertaining" only indicates that it refers to an existing arrangement and the LitePSM, but doesn't indicated in what terms. I'd suggest to put: "Approval of DAO Resolution defining the migration from the existing Coinbase Custody Agreement (PSM-USDC-A/ RWA014-A) to the new Lightweight Peg Stability Module (LITE-PSM-USDC-A/ LitePSM)."